test/unit/models/discordactions.test.js

```diff
@@ -567,4 +569,31 @@ describe("discordactions", function () {
       }
     }).timeout(10000);
   });
+
+  describe("addInviteToInviteModel", function () {
+    let addStub;
+
+    beforeEach(function () {
+      addStub = sinon.stub(discordInvitesModel, "add").resolves({ id: "invite-test-id" });
+    });
+
```

Comment:
Why are we stubbing firestore methods?

Reply:
as this is a model test and we want to stub the response coming from there, for our tests

Reply:
I don't think we should stub here, why not directly test it?

Reply:
we have been doing this for all the test in the model

Reply:
changed this as per our discussion